lib/jwt/authorizer/builder.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module JWT
+  class Authorizer
+    module Builder
+      def build(claims = {})
+        payload = default_claims.merge!(claims)
```

Comment:
You use `#merge!`. There is no need to assign it to different variable since `default_claims` is what you want after merging with claims.

Reply:
`default_claims` is not cached, so I actually have to assign it. I use `merge!` to avoid creating two Hashes (one from `default_claims` and one after `merge`)